document/js-api/index.bs

```diff
@@ -1350,13 +1349,9 @@ WebAssembly Module Records have the following methods:
     1. For each |importedName| → |importedValue| of |value|,
         1. Perform ! [$CreateDataPropertyOrThrow$](|moduleImportsObject|, |importedName|, |importedValue|).
     1. Perform ! [$CreateDataPropertyOrThrow$](|importsObject|, |key|, |moduleImportsObject|).
-1. [=Asynchronously instantiate a WebAssembly module=] |module| with imports |importsObject| and let |instancePromise| be the result.
-1. [=Upon fulfillment=] of |instancePromise| with value |instance|:
-    1. For each |name| in the [=export name list=] of |record|,
-        1. Perform ! |record|.\[[Environment]].InitializeBinding(|name|, ! Get(|instance|.\[[Exports]], |name|)).
-    1. Perform ! [=Call=](|promiseCapability|.\[[Resolve]], undefined, &laquo; undefined &raquo;).
-1. [=Upon rejection=] of |instancePromise| with reason |r|:
-    1. Perform ! [=Call=](|promiseCapability|.\[[Reject]], undefined, &laquo; |r| &raquo;).
+1. Let |instance| be the result of [=Instantiate a WebAssembly module|instantiating the WebAssembly module=] |module| with imports |importsObject|.
```

Comment:
I think we still need to be explicit about the error handling here - do we need to say that any errors get rethrown? Alternatively should we explicitly set the error on the module?

Reply:
Yes, errors here need to be rethrown. However, isn't it implicit? For example, [Instantiate a WebAssembly Module](https://webassembly.github.io/esm-integration/js-api/#instantiate-a-webassembly-module) does not explicitly rehthrow instantiation errors, that are generated in [Instantaite the core of a WebAssembly module](https://webassembly.github.io/esm-integration/js-api/#instantiate-the-core-of-a-webassembly-module).

Reply:
Right, if it's implicit that's fine - just wanted to verify that the throw completion is being properly propagated in the move from async to sync.